server/commands/write.go

```diff
@@ -63,6 +62,7 @@ func (c *WriteCommand) validateTuplesets(ctx context.Context, req *openfgapb.Wri
 	}
 
 	var authModel *openfgapb.AuthorizationModel
```

Comment:
Do we need to have this defined here now that we are using the typesystem in what follows?

Reply:
Nope, good call. I've moved into the containing block as a `authModel, err := ...` still variable initializer.